test/core/migration/TestMigration_Core_FileSystem.dyn

```diff
@@ -0,0 +1,51 @@
+    <Dynamo.Graph.Nodes.ZeroTouch.DSFunction guid="72b40f68-ac66-487f-8dad-a1ba8b779c58" type="Dynamo.Graph.Nodes.ZeroTouch.DSFunction" nickname="FilePath.Extension" x="497.2" y="458.4" isVisible="true" isUpstreamVisible="true" lacing="Shortest" isSelectedInput="False" IsFrozen="false" assembly="DSCoreNodes.dll" function="DSCore.IO.FilePath.Extension@string" />
+    <Dynamo.Graph.Nodes.ZeroTouch.DSFunction guid="10f54985-a184-4c9d-9c24-cd483c413dda" type="Dynamo.Graph.Nodes.ZeroTouch.DSFunction" nickname="FilePath.FileName" x="469.2" y="49.5999999999999" isVisible="true" isUpstreamVisible="true" lacing="Shortest" isSelectedInput="False" IsFrozen="false" assembly="DSCoreNodes.dll" function="DSCore.IO.FilePath.FileName@string,bool">
+      <PortInfo index="1" default="True" />
+    </Dynamo.Graph.Nodes.ZeroTouch.DSFunction>
+    <Dynamo.Graph.Nodes.ZeroTouch.DSFunction guid="de7087dd-810c-4aee-a5fa-82ee18a64b94" type="Dynamo.Graph.Nodes.ZeroTouch.DSFunction" nickname="FilePath.HasExtension" x="494" y="-37.6000000000001" isVisible="true" isUpstreamVisible="true" lacing="Shortest" isSelectedInput="False" IsFrozen="false" assembly="DSCoreNodes.dll" function="DSCore.IO.FilePath.HasExtension@string" />
+    <CoreNodeModels.Input.DirectoryObject guid="4408214d-da58-448e-91af-8437118fcfde" type="CoreNodeModels.Input.DirectoryObject" nickname="Directory.FromPath" x="686.8" y="282.4" isVisible="true" isUpstreamVisible="true" lacing="Disabled" isSelectedInput="False" IsFrozen="false" />
+    <CoreNodeModels.Input.FileObject guid="7b82ad75-a822-4057-bdd6-d96502ea0ed7" type="CoreNodeModels.Input.FileObject" nickname="File.FromPath" x="708.4" y="404.8" isVisible="true" isUpstreamVisible="true" lacing="Disabled" isSelectedInput="False" IsFrozen="false" />
+    <CoreNodeModels.Input.Filename guid="63ef1405-214a-444c-9f91-2d4677f70cc4" type="CoreNodeModels.Input.Filename" nickname="File Path" x="702" y="49.5999999999999" isVisible="true" isUpstreamVisible="true" lacing="Disabled" isSelectedInput="True" IsFrozen="false">
+      <System.String>No file selected.</System.String>
+    </CoreNodeModels.Input.Filename>
+    <CoreNodeModels.Input.Directory guid="470d70bb-631b-4e6e-abec-3b11bab825e1" type="CoreNodeModels.Input.Directory" nickname="Directory Path" x="699.6" y="153.6" isVisible="true" isUpstreamVisible="true" lacing="Disabled" isSelectedInput="True" IsFrozen="false">
+      <System.String>No file selected.</System.String>
+    </CoreNodeModels.Input.Directory>
+  </Elements>
+  <Connectors />
+  <Notes />
+  <Annotations />
+  <Presets />
+  <Cameras>
+    <Camera Name="Background Preview" eyeX="-17" eyeY="24" eyeZ="50" lookX="12" lookY="-13" lookZ="-58" upX="0" upY="1" upZ="0" />
+  </Cameras>
+</Workspace>
```

Comment:
This test contains all the `FileSystem` nodes from v0.9 xml and verifies none are lost or broken in the migration